app/views/application/_forfeit_game.html.erb

```diff
@@ -0,0 +1,22 @@
+<!-- Login Modal -->
+<!-- <div id="forfeitModal" class="modal fade" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h4 class="modal-title">End Game</h4>
+        <button type="button" class="close-icon modal-close" data-modal-to-close="#forfeitModal"></button>
+      </div>
+      <div class="modal-body">
+      <p>Are you sure you want to forfeit the game?</p>   
+      </div>
+      <div class="modal-footer">
+          <ul class="form-footer-links-and-statuses">
+            <div class="form-footer-status">
+            </div>
+          </ul>
+          <button id="endGame" class="custom-modal-action btn btn-default" data-ajax-method="PATCH" data-ajax-data="" data-ajax-url="/games/2/forfeit">End Game</button>
```

Comment:
Im not 100% sure how this ajax stuff works so I'm not sure if this will be a problem or not but I thought id ask
ata-ajax-url="/games/2/forfeit"
I noticed the 2 is hardcoded, so will it always change game 2?